### PR TITLE
Fix various issues in response to UUF feedback

### PR DESCRIPTION
**Pull request by gewarren**
[Hide whitespace changes](https://github.com/dotnet/dotnet-api-docs/pull/10138/files?diff=unified&w=1).

### CODE REVIEW

File: xml/System.Net/HttpWebRequest.xml [hunk 1]
```diff
@@ -2579,7 +2579,7 @@ Both <xref:System.Net.HttpWebRequest> constructors are obsolete and should not b
       </ReturnValue>
       <Docs>
         <summary>Gets or sets the default maximum length of an HTTP error response.</summary>
-        <value>The default maximum length of an HTTP error response.</value>
+        <value>The default maximum length, in kilobytes (1024 bytes), of an HTTP error response.</value>
```
[the remark this answers is not in the document]
Comment (gewarren) [reply xml/System.Net/HttpWebRequest.xml:2582]: @ManickaP Is it KB for .NET Framework? See https://github.com/microsoft/referencesource/blob/51cf7850defa8a17d815b4700b67116e3fa283c2/System/net/System/Net/HttpWebRequest.cs#L6150.

[the remark this answers is not in the document]
Comment (ManickaP) [reply xml/System.Net/HttpWebRequest.xml:2582]: I see. Checking history, this was no-op in core until recently: https://github.com/dotnet/runtime/pull/97537 and the change interprets it as byte. We should fix it in core into KB so it's the same as in framework.

[the remark this answers is not in the document]
Comment (gewarren) [reply xml/System.Net/HttpWebRequest.xml:2582]: Thanks @ManickaP. So it's okay to leave this change as-is then?

[the remark this answers is not in the document]
Comment (ManickaP) [reply xml/System.Net/HttpWebRequest.xml:2582]: Yep, we've already fixed the code.

**Review by BillWagner — APPROVED** [review]
This LGTM @gewarren 

You can :shipit: once you and the product team are ready.

**Pull request merged**